app/views/api/integrations/apicast/shared/_mapping_rule.html.erb

```diff
@@ -2,6 +2,7 @@
 
   disabled ||= !rule.new_record? && !@marked_for_update.try(:include?,rule.id.to_s)
   html_options = { :disabled => disabled , :class => 'message-when-change' }
+  index ||= false
```

Comment:
Do we need that?

Reply:
Same template is used here https://github.com/3scale/porta/pull/530/files#diff-2eeed0d10ff9fe7aedaf9123ec70b297L25 where we are not passing `index`. So when it tries to access it here: https://github.com/3scale/porta/pull/530/files#diff-8aca4cca05f925b4b00a70b406f9638dR33, it breaks. Do you think it can be better some other way? (consider this integration page is not worthy a major refactor)

Reply:
OK then you could use  index = local_assigns[:index] instead of implicit variable assignment? And that will ensure that the variable exists whatever it is. `variable ||= boolean` is kinda ugly to read :)

Reply:
indeed hehe, didn't know how to do something like that in ruby basically. thanks!